File: Python_Programming_Assignment_1/Assignment1_3.py
# 3. Write a program which contains one function named as Add() which
#    accepts two numbers from user and return addition of that two numbers.

# Input : 11 5        
# Outpu : 16

# To find addition of given numbers
def Add(No1, No2):
    result = No1 + No2
    return result

def main():
    print("Enter first number : ")
    No1 = int(input())

    print("Enter second number : ")
    No2 = int(input())

    result = Add(No1, No2)
    print("Addition is : ", result)
    
# Starter
if __name__ == "__main__":
    main()

File: Typescript_Assignment/Typescript_Assignment1/Assignment1_3.js
// 1.3 Write a typescript program which contains on efunction named as 
//     DisplayFactors. That funcion should accept one number and display
//     factors of that number.
// Input : 20
// Output : 1   2   4   5   10
function DisplayFactors(num) {
    var List = [];
    for (var i = 1; i <= num / 2; i++) {
        if (num % i == 0) {
            List.push(i);
        }
    }
    return List;
}
var No1 = 20;
var Factors = [];
Factors = DisplayFactors(No1);
console.log("Factors of " + No1 + " are : ");
console.log(Factors);
